### ISSUE TITLE
Presentation peer review

### ISSUE THREAD

**Issue by jrflemi5** [INFO-526-SU24/final-project-rahathelal#5]
The following is the peer review of the Presentation proposal by [name of team completing peer review]. The team members that participated in this review are 

- Reagan Fleming - jrflemi5

- Content: Is the research question well designed and is the data being used relevant to the research question?
The research question is well designed and thought out. The data used is definitely relevant to the research quesiton. Possible point of improvement, spend less time going over the exact structure of the data and more time on your visualizations

- Content: Did the team use appropriate visualizations and did they interpret them accurately?
The given visualizations are appropriate to the question and dataset and are well interpreted. Good use of color in visualization 2. I would have left off the N/A chart of viz3 as it doesn't add much information

- Creativity and Critical Thought: Is the project carefully thought out? Are the limitations carefully considered? Does it appear that time and effort went into the planning and implementation of the project?
The project is well thought out. You did a great job of narrowing the scope of your questions since the first peer review

- Slides: Are the slides well organized, readable, not full of text, featuring figures with legible labels, legends, etc.?
Data introduction slides are a bit wordy. Maybe just show a schema of the data instead of a printout. The visualization slides are good.

- Professionalism: How well did the team present? Does the presentation appear to be well practiced? Are they reading off of a script? Did everyone get a chance to say something meaningful about the project?
Presentation was good. Speaker was obviously well prepared and didn't seem to be reading off a script

**Comment by rahathelal** [comment]
Hi @jrflemi5,

Thanks for your valuable feedback and suggestions.

- I have now removed NA income groups from second visualization and that presents a better plot.
- For the data introduction slide, I tried several things to have the glimpse() function display only a limited amount of information, however that did not go well after publishing the document, and so I increased the width of output. But I will research on that and make required changes.

Thanks,
